static/tutorial-assets/github.js

```diff
@@ -1,58 +1,60 @@
 const passport = require('passport');
-var LinkedInStrategy = require('passport-linkedin-oauth2').Strategy;
+var GithubStrategy = require('passport-github2').Strategy;
 const loginWithIdp = require('./loginWithIdp');
 const { createIdToken } = require('../../api-util/idToken');
 
 const radix = 10;
 const PORT = parseInt(process.env.REACT_APP_DEV_API_SERVER_PORT, radix);
 const rootUrl = process.env.REACT_APP_MARKETPLACE_ROOT_URL;
-const clientID = process.env.REACT_APP_LINKEDIN_CLIENT_ID;
-const clientSecret = process.env.LINKEDIN_CLIENT_SECRET;
+const clientID = process.env.REACT_APP_GITHUB_CLIENT_ID;
+const clientSecret = process.env.GITHUB_CLIENT_SECRET;
 
 // Identity provider and identity provider client information. They should
 // match to an identity provider client "Client ID" and "IdP ID" in Console.
-const idpClientId = process.env.LINKEDIN_PROXY_CLIENT_ID;
-const idpId = process.env.LINKEDIN_PROXY_IDP_ID;
+const idpClientId = process.env.GITHUB_PROXY_CLIENT_ID;
+const idpId = process.env.GITHUB_PROXY_IDP_ID;
 
 let callbackURL = null;
 
 const useDevApiServer = process.env.NODE_ENV === 'development' && !!PORT;
```

Comment:
Hmm, I'm wondering why the callbackUrl would ever be on localhost, if the proxy set up doesn't work with localhost at all

Reply:
But no need to dig into this if there's no obvious answer, I realize that this is old code

Reply:
At least I tested the Github auth part on localhost before deploying it to Heroku for testing towards the Sharetribe APIs, and I guess this is generally a remnant of the non-proxy auth processes that do work locally. So I'd leave it in just in case folks do want to do this part of developing the feature locally.